nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/operations/document/FilterDocuments.java

```diff
@@ -73,16 +74,19 @@
             DocumentAttributeFilterFactory.IMMUTABLE_DOC, DocumentAttributeFilterFactory.MUTABLE_DOC })
     protected String attr;
 
-    @OperationMethod
-    public DocumentModelList run(DocumentModelList docs) {
+    /**
+     * Method rewritten to use Collector in order to execute condition on each document instead of the whole input list
```

Comment:
That's an interesting implementation note but it does not belong the the Javadoc. Put it in a normal comment before or in the method.